api-server.py

```diff
@@ -101,9 +118,37 @@ def main():
     elif args.coznet:
         settings.setup_coznet()
 
+    syslog_facility = None
+    if args.syslog or args.syslog_local:
```

Comment:
I had originally made `--syslog-local` only apply if you also applied `--syslog` intentionally, but this makes it a bit cleaner to use in that you can use one or the other (and it still won't hurt to use both) 👍

Reply:
I think we take either this way, or show an error if only using `--syslog-local` and not `--syslog`

Reply:
I think we can leave it and err on the side of flexibility.